### PR TITLE
Carousel - enablePagination

### PR DESCRIPTION
**Pull request by Inbal-Tish**
Adding 'enablePagination' prop to allow blocking multiple pages scroll (full-width pages only)

### CODE REVIEW

File: demo/src/screens/componentScreens/CarouselScreen.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ class CarouselScreen extends Component {
         </View>
 
         <View padding-20>
-          <Carousel migrate containerStyle={{height: 160}} initialPage={INITIAL_PAGE} loop>
+          <Carousel migrate containerStyle={{height: 160}} initialPage={INITIAL_PAGE} loop usePagination>
```
Comment (ethanshar) [inline demo/src/screens/componentScreens/CarouselScreen.js]: what's `usePagination`? I don't you defined this prop anywhere..

File: src/components/carousel/index.js [hunk 2]
```diff
@@ -164,6 +172,11 @@ export default class Carousel extends BaseComponent {
     return !loop && pageWidth;
   }
 
+  shouldEnablePagination() {
```
Comment (ethanshar) [inline src/components/carousel/index.js]: I don't think the user needs to control if the paging is enabled or not. 
in a full-width page carousel it should always come with paging. 
I think it's a bug, not a feature..

File: demo/src/screens/componentScreens/CarouselScreen.js (hunk 1 above)
Comment (Inbal-Tish) [reply demo/src/screens/componentScreens/CarouselScreen.js]: opps. Fixed

File: src/components/carousel/index.js (hunk 2 above)
Comment (Inbal-Tish) [reply src/components/carousel/index.js]: Why? In ScrollView the user controls it, why not in the Carousel? We can default it to true and the user can disable it...

Comment (ethanshar) [reply src/components/carousel/index.js]: Cause the whole point of Carousel is the paging. 
Anyway, I'm fine with setting the default value to true.

Comment (Inbal-Tish) [reply src/components/carousel/index.js]: Done.

**Pull request merged**